ProjectGroupA/src/main/java/com/fundation/search/controller/Controller.java

```diff
@@ -23,13 +23,13 @@
  */
 public class Controller {	
```

Comment:
If I'm not mistaken the whole methods to read values should not be available for the merge, otherwise they would continue work independently... right?
getFileName
getExtension
getSize
getOperator
displayResults